community/collections/src/main/java/org/neo4j/collection/PrimitiveLongArrayQueue.java

```diff
@@ -45,7 +45,10 @@ public PrimitiveLongArrayQueue()
 
     PrimitiveLongArrayQueue( int capacity )
     {
-        assert (capacity != 0) && ((capacity & (capacity - 1)) == 0) : "Capacity should be power of 2.";
+        if ( capacity == 0 || (capacity & (capacity - 1)) != 0 )
+        {
+            throw new IllegalArgumentException( "Capacity should be power of 2." );
```

Comment:
maybe add actual value to the error message?

Reply:
yes updated